<launch>
	<!-- Launch serialOutput node -->
	<node pkg="urGovernor" type="serialOutput" name="serialOutput" output="screen">
		<rosparam command="load" file="$(find urGovernor)/config/governor.yaml" />
	</node>

	<!-- Launch kinematicsTest node -->
	<node pkg="urGovernor" type="kinematicsTest" name="kinematicsTest" output="screen">
		<rosparam command="load" file="$(find urVision)/config/common.yaml" />
		<rosparam command="load" file="$(find urGovernor)/config/governor.yaml" />
        </node>
</launch>
